glossary.yml

```diff
@@ -2505,6 +2505,12 @@
       thalela (`_`), khutlo (`.`) kana thaladi (`-`) mo maineng a maemedi a boleng jo bo fetogang (variables) kana a dilo (objects). 
       Mo godimo ga moo, lefoko lengwe le lengwe mo maineng a le simolola ka tlhakakgolo. 
       Dikai tsa mokwalo o ke `CalculateSum` le `findPattern`.
+  uk:
+    term: "camel case (верблюдячий регістр)"
+    def: >
+      Стиль написання коду, який передбачає іменування змінних та об'єктів без пробілів, 
+      символів підкреслення (`_`), крапки (`.`) або тире (`-`), а кожне слово пишеться
+      з великої літери. Наприклад: `ЗнайтиСумму` і `знайтиЗразок`.
```

Comment:
Do not translate `CalculateSum` and `findPattern` - you wouldn't name them in Ukrainian in your code.

Reply:
@olexandr-konovalov , I agree, updated